adaptive/build.gradle.kts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@file:Suppress("UnstableApiUsage")
+
+plugins {
+    id("com.android.library")
```

Comment:
Let's add plugins to catalog as well - https://github.com/google/accompanist/blob/main/gradle/libs.versions.toml, here is how https://developer.android.com/build/migrate-to-catalogs#migrate-plugins

Reply:
I tried this, but since we are on agp version 7.4.1 I think we have this issue (https://github.com/gradle/gradle/issues/17968) since we have to maintain the class path version as well. I can do a workaround like this
`id(libs.plugins.android.library.get().pluginId)` would that be okay?

Reply:
Yes please, as we continue to enjoy the catalogs, and we can change this code once the Gradle bug is fixed

Reply:
On the same thought we should move to AGP 8.0 which is stable

Reply:
Added this in 6bbdef1 :) Moving to AGP 8.0, should that maybe be in a separate PR if these are squashed in case it breaks something? Wdyt? :)

Reply:
@bentrengrove call :) (my opinion makes sense to do in a separate PR)

Reply:
Yes, let's upgrade. Separate PR please, if you have time to do it that would be really great to unblock this. But if not, definitely no problem. It will just take me ~1 week to get to it.

Reply:
I can take a look into upgrading to AGP 8.0 parallel to this :)

Reply:
Added an attempt on this here: https://github.com/google/accompanist/pull/1590

Reply:
Does this now solve the need for using pluginId?

Reply:
I tried to rebase this locally and upgrading does not fix it. I think the fix is to migrate all the plugins at classpath to use the alias and then update this to use alias as well. The reason I think this is that the sync error is that it already has this plugin on the classpath with an unknown version, so the compatibility cannot be checked.

I can fix it if I add the `pluginManagment` block with `repositories` block to `settings.gradle` and then remove the classpath decleration and move the plugins to the `plugins` block using the `alias` there as well in the project gradle file. Should I do that here? Or wait until I migrate the project gradle and settings gradle file to Kotlin DSL? Or I can do it in a seperate PR if you want to seperate these two things.

Reply:
Let's wait until you migrate those other files and go with the current solution for now. I will merge this one in!